### PR TITLE
kubeadm cleanup: master -> control-plane :)

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
cleanup: master -> control-plane

**Which issue(s) this PR fixes**:
Refs kubernetes/kubeadm#1376

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/etcd/local_test.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func TestGetEtcdPodSpec(t *testing.T) {
-	// Creates a Master Configuration
+	// Creates a Control Plane Configuration
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/etcd/local_test.go]: `...ClusterConfiguration`

File: cmd/kubeadm/app/phases/kubeconfig/doc.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ package kubeconfig
 
 	INPUTS:
 		From InitConfiguration
-			The Master API Server endpoint (AdvertiseAddress + BindPort) is required so the kubeconfig file knows where to find the master
+			The Control Plane API Server endpoint (AdvertiseAddress + BindPort) is required so the kubeconfig file knows where to find the control-plane
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/doc.go]: `The API Server endpoint...`

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go [hunk 3]
```diff
@@ -45,7 +45,7 @@ func TestGetKubeConfigSpecsFailsIfCADoesntExists(t *testing.T) {
 	tmpdir := testutil.SetupTempDir(t)
 	defer os.RemoveAll(tmpdir)
 
-	// Creates a Master Configuration pointing to the pkidir folder
+	// Creates a Control Plane Configuration pointing to the pkidir folder
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go]: `Creates a ClusterConfiguration...`

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go [hunk 4]
```diff
@@ -66,7 +66,7 @@ func TestGetKubeConfigSpecs(t *testing.T) {
 	// Adds a pki folder with a ca certs to the temp folder
 	pkidir := testutil.SetupPkiDirWithCertificateAuthorithy(t, tmpdir)
 
-	// Creates Master Configurations pointing to the pkidir folder
+	// Creates Control Plane Configurations pointing to the pkidir folder
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go]: `Creates a ClusterConfiguration...`

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go [hunk 5]
```diff
@@ -305,7 +305,7 @@ func TestCreateKubeconfigFilesAndWrappers(t *testing.T) {
 		// Adds a pki folder with a ca certs to the temp folder
 		pkidir := testutil.SetupPkiDirWithCertificateAuthorithy(t, tmpdir)
 
-		// Creates a Master Configuration pointing to the pkidir folder
+		// Creates a Control Plane Configuration pointing to the pkidir folder
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go]: `Creates a ClusterConfiguration...`

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go [hunk 6]
```diff
@@ -335,7 +335,7 @@ func TestWriteKubeConfigFailsIfCADoesntExists(t *testing.T) {
 	tmpdir := testutil.SetupTempDir(t)
 	defer os.RemoveAll(tmpdir)
 
-	// Creates a Master Configuration pointing to the tmpdir folder
+	// Creates a Control Plane Configuration pointing to the tmpdir folder
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go]: `Creates a ClusterConfiguration...`

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go [hunk 7]
```diff
@@ -382,7 +382,7 @@ func TestWriteKubeConfig(t *testing.T) {
 		t.Fatalf("couldn't retrieve ca cert: %v", err)
 	}
 
-	// Creates a Master Configuration pointing to the pkidir folder
+	// Creates a Control Plane Configuration pointing to the pkidir folder
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig_test.go]: `Creates a ClusterConfiguration...`

**Review by neolit123 — COMMENTED** [review]
thanks @SataQiu 
added some minor comments.

please remember to squash the commits to one.

**Review by neolit123 — APPROVED** [review]
/lgtm
/approve
thanks!

**Comment by BenTheElder** [comment]
/hold
@neolit123 to unhold
"Fixes <issue>" in the PR description will close the issue even though this is not everything. I forget if only the commit description does this, but it might also close it when merged into various kubernetes forks.

**Comment by neolit123** [comment]
yes, @SataQiu 
please use:
`refs kubernetes/kubeadm#1376` instead of `fixes`

**Comment by SataQiu** [comment]
OK! @neolit123 
I changed the description.

**Pull request merged**